custom_components/unifiprotect/config_flow.py

```diff
@@ -92,17 +92,13 @@ async def async_step_user(self, user_input=None):
                 CONF_PORT: user_input[CONF_PORT],
                 CONF_USERNAME: user_input.get(CONF_USERNAME),
                 CONF_PASSWORD: user_input.get(CONF_PASSWORD),
-                CONF_DISABLE_RTSP: user_input.get(CONF_DISABLE_RTSP),
-                CONF_DOORBELL_TEXT: user_input.get(CONF_DOORBELL_TEXT),
-                CONF_SNAPSHOT_DIRECT: user_input.get(CONF_SNAPSHOT_DIRECT),
-                CONF_SCAN_INTERVAL: DEFAULT_SCAN_INTERVAL,
             },
             options={
                 CONF_USERNAME: user_input.get(CONF_USERNAME),
                 CONF_PASSWORD: user_input.get(CONF_PASSWORD),
-                CONF_DISABLE_RTSP: user_input.get(CONF_DISABLE_RTSP),
-                CONF_DOORBELL_TEXT: user_input.get(CONF_DOORBELL_TEXT),
-                CONF_SNAPSHOT_DIRECT: user_input.get(CONF_SNAPSHOT_DIRECT),
+                CONF_DISABLE_RTSP: False,
+                CONF_DOORBELL_TEXT: "",
+                CONF_SNAPSHOT_DIRECT: False,
```

Comment:
We need to migrate the data in `__init__.py` `async_setup_entry` using `async_update_entry` to move it from `data` into `options` and then change the places where the data is looked up in the integration

There is an example in the yeelight integration see `_async_normalize_config_entry`

Reply:
Thanks. I will have a look and make the change.